### PR TITLE
[I18n support] Maybe I can make some contribution

### PR DESCRIPTION
**Pull request by jianzhichun**
## Description
Hello budibase team, this is Zao(早). I have made some i18n support for budibase and I am not sure if it's helpful. 
I am working on low-code (or called hpapass) two years at alibaba.inc. I like this project and it will be my honer to be one of the participants.

By the way, I have found some confusion about multi-tenant feature and oauth callback on non-multi-tenant situation so I also made some commits digress. (e.g. https://github.com/Budibase/budibase/pull/2543/files#diff-239787aac07e2f19359280c20e881c06acf147a8a796fe1f3a63b9d5a21cd5f2R28)

## Screenshots
image
image
image

### CODE REVIEW

**Comment by jianzhichun** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by jianzhichun** [comment]
recheck

File: packages/builder/src/App.svelte [hunk 1]
```diff
@@ -6,8 +6,20 @@
   import { NotificationDisplay } from "@budibase/bbui"
   import { parse, stringify } from "qs"
   import HelpIcon from "components/common/HelpIcon.svelte"
+  import { initI18n } from "builderStore/store/i18n.js"
+  import { organisation } from "./stores/portal"
+
+  initI18n()
 
   onMount(async () => {
+    if (!$organisation.logoUrl) {
```
Comment (shogunpurple) [inline packages/builder/src/App.svelte:15]: This is nice - cool to have your company name and logo favicon in the title 👍

**Review by shogunpurple — COMMENTED** [review]
Hi @jianzhichun, first of all thank you so much for putting together this PR and for the clear amount of time and effort you have put into it. The i18n support looks great, and is set up in a sensible way. 

There's a few things here that we would need to address before moving ahead.

#### Private NPM Registry
There are a few private NPM registry references throughout this PR, and dev versions have been published to it. We would need to remove all of these, so that budibase continues to release to our public NPM repository.

#### Locale Switcher
Without using the Locale switcher, will it respect the locale of the users browser? This is important for most users.

Once these are addressed - this is such a huge change, so it will need to go through quite a bit of testing end to end before merging.

Thanks again for taking the time to do this, we appreciate your effort and will work with you to get this merged into the platform after feedback.

**Pull request closed without merging**